### ISSUE TITLE
No Mailbox in Light's Hope Chapel entrance.

### ISSUE THREAD

**Issue by kaytotes** [Project-Epoch/bugtracker#5811]
> No Mailbox in Light's Hope Chapel entrance.

.go xyz 2282.800000 -5318.310000 88.669600 0 5.335350

**Comment by zana244** [comment]
Correct, there is no mailbox on internal at LHC. However, it was only added in vanilla on patch 1.11 when they introduced Naxxramas. Up to devs if they want it.
